packages/react/src/codemods/constants.tsx

```diff
@@ -1,5 +1,18 @@
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-const pkgJson = require('../../package.json');
+/**
+ * Because we don't want to include package json in the build artefacts
+ * AND because the location of package json will be different depending
+ * on using source or built assets - we do this. Not great but for now
+ * it's better than re-writing the build setup.
+ */
+const tryGetPkgJson = () => {
```

Comment:
this is shit but it works - without including package json into the built assets (changing the TS build to have ./ as base path instead of ./src i don't know if we can stabilize this.

my preference is to make this fix asap, and maybe re-evaluate build setup later.